fastlane_bot/bot.py

```diff
@@ -949,6 +965,8 @@ def _handle_trade_instructions(
             trade_instructions=ordered_trade_instructions_objects
         )
 
+        # ordered_trade_instructions_objects[0].db = QueryInterface(ConfigObj=self.ConfigObj)
+
```

Comment:
Seems like a redundant change

Reply:
not sure what you're referring to here

Reply:
This is new code (painted green), right?
And it is masked out, right?
So that makes it a redundant change as far as I can say (did you forget to remove it?).

Reply:
got it. fixed, thanks